### PR TITLE
[RPGF-44] Explore Page, Hamburger Typo, Design

### PR DESCRIPTION
**Pull request by Mark169Z**
# Description

[//]: # "Please include a summary of the changes and the related issue. (You can write below without deleting this comment)"
Fix the CSS bugs and design issues on the Explore pages, and correct the typo on the hamburger menu.
### What
[//]: # "Briefly describe this feature / fix / refactor (You can write below without deleting this comment)"
- Explore Page padding changed on Mobile
- OP Display on Retro Funding 4 Page
- Fix the 'See All' button for Retro Funding 1 not showing on mobile devices on the Explore Page.
- Change Typo "SignIn" to "Sign In" in Hamburger menu
- Fix avatar overflow in DynamicCard
- Fix 'See All' button breakpoint issue
- Fix 'See All' button for Retro Funding 2 on mobile not working as intended
- Add ranking on Retro Funding Page 1,2,4 (Need rank param)
- Logo Image Size in DrawerNav
- Retro Funding 4 in Explore page design change
- Retro Funding 2 in Explore page design change (Rounded consistency)
- Project Detail Page Design change
### Why
[//]: # "Describe the reason why we need this feature / fix / refactor or refer to Jira Ticket (You can write below without deleting this comment)"
- To prevent unexpected issues that can occur on some pages
- To improve design consistency

Jira Ticket: RPGF-44
## Type of change
Please check the relevent option with `x`.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if needed)
- [x] My changes generate no new warnings
- [x] I have tested on my `Local Machine`

### CODE REVIEW

**Comment by wit03** [comment]
I'll be reviewing this PR tonight

**Comment by Billy19191** [comment]
Also, do it this round (round 4) for design consistency, showing awards like this. @wit03 assign this task to you now, good luck!

Screenshot 2567-09-11 at 23 30 17

File: app/(white_navbar)/explore/RetroPGF1/_component/DynamicCard.tsx [hunk 1]
```diff
@@ -45,22 +48,23 @@ return (
...
         {/* Avatar */}
-        <div className="absolute top-4 left-4 rounded-[0.25rem] flex flex-shrink-0 z-20">
+        <div className="absolute top-10 lg:top-12 left-4 rounded-[0.25rem] bg-white flex flex-shrink-0 z-20 overflow-hidden">
             <Image
                 src={avatar || "/random/avatar.png"}
                 alt="avatar image"
                 className=""
                 width={48}
                 height={48}
+                style={{ width: "48px", height: "48px" }}
```
[the remark this answers is not in the document]
Comment (Mark169Z) [reply app/(white_navbar)/explore/RetroPGF1/_component/DynamicCard.tsx:58]: Changed

File: app/(white_navbar)/explore/RetroPGF5/_component/DynamicCard.tsx [hunk 2]
```diff
@@ -78,6 +78,7 @@ export default function DynamicCard({
           className=""
           width={48}
           height={48}
+          style={{ width: "48px", height: "48px" }}
```
[the remark this answers is not in the document]
Comment (Mark169Z) [reply app/(white_navbar)/explore/RetroPGF5/_component/DynamicCard.tsx:81]: Changed

File: app/(white_navbar)/explore/RetroPGF4/_component/DynamicCard.tsx [hunk 3]
```diff
@@ -82,6 +84,7 @@ export default function DynamicCard({
           className=""
           width={48}
           height={48}
+          style={{ width: "48px", height: "48px" }}
```
[the remark this answers is not in the document]
Comment (Mark169Z) [reply app/(white_navbar)/explore/RetroPGF4/_component/DynamicCard.tsx:87]: Changed

File: app/(white_navbar)/explore/RetroPGF4/_component/Tab/BadgeholderTab.tsx [hunk 4]
```diff
@@ -27,7 +27,7 @@ const BadgeholderTab: FC<BadgeholderTabProps> = ({ badgeholderData }) => {
     filter: true,
   })
   const [checkBox, setCheckBox] = useState<CheckBoxStateBadgeholderType>({
-    status: 'All',
```
[the remark this answers is not in the document]
Comment (Mark169Z) [reply app/(white_navbar)/explore/RetroPGF4/_component/Tab/BadgeholderTab.tsx:30]: Changed

**Comment by Billy19191** [comment]
I will review these things within 1-2 days. Sorry for the take time ja.

File: app/(white_navbar)/explore/RetroPGF4/_component/DynamicCard.tsx [hunk 5]
```diff
@@ -103,16 +105,22 @@ export default function DynamicCard({
...
-              <p className="text-sm font-light text-gray-600">
-                <Image src={OpTokenPath} alt="OpToken" width={18} height={18} />
+              <p className="text-sm font-light text-gray-600">People</p>
+            </div>
+          </div>
+          <div className="flex gap-2">
+            <Trophy size={20} />
+            <div className="flex gap-1">
+              <p className="text-sm font-semibold text-gray-800">
+              {reward!= -1 ? numberWithCommas(reward.toFixed(2)||0) : 0} OP
```
[the remark this answers is not in the document]
Comment (wit03) [reply app/(white_navbar)/explore/RetroPGF4/_component/DynamicCard.tsx:121]: @Mark169Z  It is better here to just edit the data in JSON file with `CTRL + F` and replace -1 with 0 so we don't have to waste performance on this filtering.

File: app/(white_navbar)/explore/RetroPGF4/RetroType4.d.ts [hunk 6]
```diff
@@ -82,6 +83,7 @@ export interface iRetroPGF4Project {
   impactMetrics: ImpactMetrics | null
   isOpenSource: boolean
   reward?: number
+  ranking?: number
```
[the remark this answers is not in the document]
Comment (Billy19191) [reply app/(white_navbar)/explore/RetroPGF4/RetroType4.d.ts:86]: Ignore this, ranking is fine 💯

**Review by Billy19191 — APPROVED** [review]
LGTM Eiei 🥇

**Pull request merged**